ansible/roles/csm.sbps.mount_s3_images/README.md

```diff
@@ -0,0 +1,37 @@
+csm.sbps.mount_s3_images
+========================
+
+Mount s3 bucket 'boot-images' using s3fs read-only policy for SBPS agent
+
+Requirements
+------------
+
+None.
+
+Role Variables
+--------------
+
+Dependencies
+------------
+
+Example Playbook
+----------------
+
+```yaml
+    - hosts: Iscsi_Targets
```

Comment:
Should this be `Iscsi_Targets` or `Management_Worker` ?
If this is supposed to run on all NCN workers and nowhere else, then shouldn't we use the latter?